### PR TITLE
some fixes plus suse support

### PR DESCRIPTION
**Pull request by jgrumboe**
[fix] refix the issue #60, was reintroduced with 3de3773
[fix] always use full path to binaries
[new] added rhn_register support for use with suse manager 1.7/2.1

### CODE REVIEW

**Comment by jgrumboe** [comment]
Could someone have a look into this pull? Thx :)

File: plugins/guests/redhat/cap/rhn_register.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@ def self.rhn_register(machine)
         def self.rhn_register_register(machine, ui)
           rhn_register_upload_certificate(machine, ui)
           rhn_register_server_url(machine, ui) if machine.config.registration.serverurl
-          command = "rhnreg_ks #{configuration_to_options(machine.config.registration)}"
+          command = "/usr/sbin/rhnreg_ks #{configuration_to_options(machine.config.registration)}"
```
Comment (LalatenduMohanty) [inline plugins/guests/redhat/cap/rhn_register.rb]: Is there any specific reason we are using a absolute path here?

**Comment by LalatenduMohanty** [comment]
@whitel @pvalena @strzibny ^^

@jgrumboe can we please have separate PRs specific to SUSE and RHEL?

**Comment by jgrumboe** [comment]
yes, i will do so, need some minutes/hours ;)

File: plugins/guests/redhat/cap/rhn_register.rb (hunk 1 above)
Comment (jgrumboe) [inline plugins/guests/redhat/cap/rhn_register.rb]: cz with "sudo" (for example as vagrant-user) "rhnreg_ks" is not in the path of the vagrant-user.
With full path it works perfectly for root directly and vagrant via sudo - at least for me.

**Comment by jgrumboe** [comment]
splitted into #67 und #68

**Pull request closed without merging**